install.sh

```diff
@@ -712,16 +712,16 @@ additional_shellenv_commands=()
 if [[ "${HOMEBREW_BREW_DEFAULT_GIT_REMOTE}" != "${HOMEBREW_BREW_GIT_REMOTE}" ]]
 then
   ohai "HOMEBREW_BREW_GIT_REMOTE is set to a non-default URL:"
-  echo "${tty_underline}${HOMEBREW_BREW_GIT_REMOTE}${tty_reset} will be used for Homebrew/brew Git remote."
```

Comment:
The phrase was taken from:

https://github.com/Homebrew/brew/blob/aaea979a428a004a9a681e92deecc122320c5a48/Library/Homebrew/cmd/update.sh#L495-L497

Reply:
Thanks, I've made a note to update those as well.